### ISSUE TITLE
inserting length with "values" field for continuous datatype returns missing value_ids error, worked in v2, fails in v3

### ISSUE THREAD

**Issue by alihammadraza** [etsy/open-api#659]
### Discussed in https://github.com/etsy/open-api/discussions/658

<div type='discussions-op-text'>

<sup>Originally posted by **alihammadraza** November 14, 2022</sup>
While trying to PUT length property id 47626759838 for a listing with taxonomy id 929 I get error message saying "Missing input parameter: [value_ids]". This is a continuous property as in the length can be anything in inches and therefore should not require value_ids. The input fields I am using are as follows:
<pre>
(
    [scale_id] => 5
    [values] => 42
    [scale_name] => Inches
)
</pre>

Excerpt below is direct from taxonomy 929's details for Length :
<pre>
{
    property_id":47626759838,
    "name":"Length",
    "display_name":"Length",
    "scales":[
        {
            "scale_id":7,
            "display_name":"Centimeters",
            "description":""
        },
        {
            "scale_id":11,
            "display_name":"Feet",
            "description":""
        },
        {
            "scale_id":5,
            "display_name":"Inches",
            "description":""
        },
        {
            "scale_id":10,
            "display_name":"Meters",
            "description":""
        },
        {
            "scale_id":4,
            "display_name":"Millimeters",
            "description":""
        },
        {
            "scale_id":9,
            "display_name":"Yards",
            "description":""
        }
    ],
    "is_required":false,
    "supports_attributes":true,
    "supports_variations":true,
    "is_multivalued":false,
    "max_values_allowed":5,
    "possible_values":[
    ],
    "selected_values":[
    ]
}
</pre>
<br/>

When we GET properties for a similar listing filled using web UI, the length property is filled and the value_ids comes back with a huge number in billions. I presume etsy maintains a data set of all continuous values versus generated value_ids. However, I do not have access to any existing value_ids when I use the api to PUT length property in inches for a new listing. 
<br/>

I have tried random large numbers as value_ids which it seems does get stored somewhere in etsy since a GET request for properties against that listing comes back with the previously inserted values including both value and value_ids. However, the shop's web ui doesn't show the length despite api results.

<br/>

Simply put: value_ids should not be necessary for insertion in a listing like handmade rugs which doesn't come in specific sizes. This is a bug in v3. The length attribute (with same id) in api v2 worked just fine for the exact taxonomy without requiring value_ids.</div>

**Comment by nateswart** [comment]
Please note that the Github Issue Queue is being decommissioned in favor of [Github Discussions](https://github.com/etsy/open-api/discussions). New issues in the issue queue are no longer being accepted.
